### PR TITLE
Fixed tooltip size, now it is not overlaped by toolbar.

### PR DESCRIPTION
**Pull request by bond95**
Fixes: https://github.com/oVirt/ovirt-web-ui/issues/514

image

---
This change is [Reviewable](https://reviewable.io/reviews/ovirt/ovirt-web-ui/520)

### CODE REVIEW

**Comment by mareklibra** [comment]
Really scary. Based on experimental feature (expected to be changed), not supported in IE or Edge.
I'm not strictly against it but what are the options to do it in a production-alligned way?

**Comment by jniederm** [comment]
Here is why I believe we can stay calm and use `max-content`:
* The table has been updated (not by me).
screenshot-developer mozilla org-2018-03-14-19-35-51
* It works (tested on latest chrome and firefox)
* We use css autoprefixer so it's supposed to work even several versions back
* If the style definition is turned of in dev tools, it, the popup is shifted. It doesn't look nice, but it's fully functional

Aternatives I can see:
* Set fixed, large enough width. It would look ugly for short tooltips.
* Guestimate width based on length of tooltip string - requires computation, unreliable.

**Comment by mareklibra** [comment]
Support of IE/Edge is important.
Simple setting of `z-index` did not help (tried some time ago).
The order of components should be changed to properly fix the issue.

In the meantime, since the tooltip component is local to this project and we don't have tons of tooltips, I suggest to introduce new optional parameter denoting width of the component (via appending className).

**Comment by bond95** [comment]
@mareklibra please check.

File: src/components/FieldHelp/index.js [hunk 1]
```diff
@@ -39,6 +40,7 @@ FieldHelp.propTypes = {
   tooltip: PropTypes.string,                                          // tooltip shown when hovering the text
   children: PropTypes.any,
   container: PropTypes.any,
+  popoverWidth: PropTypes.string,
```
Comment (mareklibra) [inline src/components/FieldHelp/index.js]: Can you please add a comment here, that CSS's `width: max-content` is not supported in IE/Edge and this is a work-around for special cases?

**Comment by mareklibra** [comment]
Will do follow-up, new release is awaiting.

**Pull request merged**